Cargo.toml

```diff
@@ -290,9 +290,10 @@ hostname = "0.4"
 indicatif = "0.17.8"
 itertools = "0.13.0"
 libc = "0.2.153"
-lscolors = { version = "0.16.0", default-features = false, features = [
+# this is just for a draft pr
+lscolors = { default-features = false, features = [
   "gnu_legacy",
-] }
+], git = "https://github.com/matrixhead/lscolors" }
```

Comment:
is it going to be hard to merge your changes upstream ?

Reply:
i'm not exactly sure, let me put together a pr on the upstream repo and get a review from them